io.catenax.pcf/5.0.0/Pcf.ttl

```diff
@@ -0,0 +1,903 @@
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :ProductIdsTrait .
+
+:productCategoryCpc a samm:Property ;
+   samm:preferredName "Product Category"@en ;
+   samm:description "Mandatory: UN (United Nations) Product Classification Code (CPC - Central Classification Code) of a given product as specified the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. WBCSD specific extension, which will probably be declared as \"optional\" in a later WBCSD specification version. In Catena-X for example specified with default value \"011-99000\"."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "011-99000" .
+
+:productNameCompany a samm:Property ;
+   samm:preferredName "Product Trade Name"@en ;
+   samm:description "Mandatory: Non-empty trade name of a product as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :NonEmptyStringTrait ;
+   samm:exampleValue "My Product Name" .
+
+:pcf a samm:Property ;
+   samm:preferredName "PCF (Product Carbon Footprint)"@en ;
+   samm:description "A PCF (Product Carbon Footprint) represents the carbon footprint of a product and related data as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :CarbonFootprint .
+
+:pcfLegalStatement a samm:Property ;
+   samm:preferredName "PCF (Product Carbon Footprint) Legal Statement"@en ;
+   samm:description "Optional: Option for legal statement/ disclaimer as specified in the Catena-X PCF Rulebook."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "This PCF (Product Carbon Footprint) is for information purposes only. It is based upon the standards mentioned above." .
+
+:IdTrait a samm-c:Trait ;
+   samm:preferredName "ID (Identifier) Trait"@en ;
+   samm:description "Trait for defining a PCF (Product Carbon Footprint) identifier as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm-c:baseCharacteristic :PfId ;
+   samm-c:constraint :UUIDv4 .
```

Comment:
Here you can make use of existing characteristic and don't need to define a new one.
For reference: https://github.com/eclipse-tractusx/sldt-semantic-models/pull/405/files#diff-8294717bc9be9d276f548ae9f83e2831503589ef17d4d5bf6f1463cdb397d00eR45

Reply:
@agg3fe ; @jacewski-bosch : changed to use existing shared characteristic and removed redundant traits/ constraints. Please check again. As already mentioned i would like to achieve MS2 and MS3 during our next regular call on Dec 11th 2023. Thanks in advance.